category/traversable/lemmas.lean

```diff
@@ -0,0 +1,65 @@
+
+import tactic.cache
+import category.traversable.basic
+
+universe variables u
+
+open is_lawful_traversable
+open function (hiding comp)
+open functor
+
+attribute [functor_norm] is_lawful_traversable.naturality
+
+namespace traversable
+
+variable {t : Type u → Type u}
+variables [traversable t] [is_lawful_traversable t]
+variables {G H : Type u → Type u}
+
+variables [applicative G] [is_lawful_applicative G]
+variables [applicative H] [is_lawful_applicative H]
+variables {α β γ : Type u}
+variables g : α → G β
+variables h : β → H γ
+variables f : β → γ
+
+variables G H
+
+def pure_transformation : applicative_transformation id G :=
+begin
+  refine { F := @pure G _, .. } ;
+  intros, { refl }, { simp, refl }
```

Comment:
This proof is hard to read. Only the `intros` is applied to all new goals. I would put `intros` at the end of the previous line and the `{ .. }` parts on separate lines.